common/build.gradle.kts

```
@@ -104,4 +107,10 @@ val packForXCode by tasks.creating(Sync::class) {
     }
 }
 
-tasks.getByName("build").dependsOn(packForXCode)
\ No newline at end of file
+tasks.getByName("build").dependsOn(packForXCode)
+
+sqldelight {
+    database("exampleDB") { // This will be the name of the generated database class.
```

```suggestion
    database("Database") {
```

We can just use this name since we'll only have one database in the app